terraform/gcp/README.md

```diff
@@ -0,0 +1,259 @@
+
+Next, let's link a billing account to that project. To determine what billing accounts are available, run the following command:
+
+```console
+gcloud alpha billing accounts list
+```
+
+Then set the billing account ID `GOOGLE_BILLING_ACCOUNT` environment variable:
+
+```console
+export GOOGLE_BILLING_ACCOUNT="XXXXXXX"
+```
+
+So we can link the `GOOGLE_BILLING_ACCOUNT` with the previously created `GOOGLE_PROJECT`:
+
+```console
+gcloud alpha billing projects link "$GOOGLE_PROJECT" --billing-account "$GOOGLE_BILLING_ACCOUNT"
+```
+
+### Enable Compute API
+
+In order to deploy VMs to the project, we need to enable the compute API:
+
+```console
+gcloud services enable compute.googleapis.com
+```
+
+### Create Terraform Service Account
+
+Finally, let's create a Terraform Service Account user and its `account.json` credentials file:
+
+```console
+gcloud iam service-accounts create terraform \
+    --display-name "Terraform Service Account" \
+    --description "Service account to use with Terraform"
+```
+
+```console
+gcloud projects add-iam-policy-binding "$GOOGLE_PROJECT" \
+  --member serviceAccount:"terraform@$GOOGLE_PROJECT.iam.gserviceaccount.com" \
+  --role roles/editor
+```
+
+```console
+gcloud iam service-accounts keys create account.json \
+    --iam-account "terraform@$GOOGLE_PROJECT.iam.gserviceaccount.com"
+```
+
+> ⚠️ **Warning**
+>
+> The `account.json` credentials gives privileged access to this GCP project. Be careful to avoid leaking these credentials by accidentally committing them to version control systems such as `git`, or storing them where they are visible to others. In general, storing these credentials on an individually operated, private computer (like your laptop) or in your own GCP cloud shell is acceptable for testing purposes. For production use, or for teams, use a secrets management system like HashiCorp [Vault](https://www.vaultproject.io/) For this tutorial's purposes, we'll be storing the `account.json` credentials on disk in the cloud shell.
+
+Now set the *full path* of the newly created `account.json` file as `GOOGLE_APPLICATION_CREDENTIALS` environment variable.
+
+```console
+export GOOGLE_APPLICATION_CREDENTIALS=$(realpath account.json)
+```
+
+### Ensure Required Environment Variables Are Set
+
+Before moving onto the next steps, ensure the following environment variables are set:
+
+* `GOOGLE_PROJECT` with your selected GCP project name.
+* `GOOGLE_APPLICATION_CREDENTIALS` with the *full path* to the Terraform Service Account `account.json` credentials file created in the last step.
+
+## Build HashiStack Golden Image with Packer
+
+[Packer](https://www.packer.io/intro/index.html) is HashiCorp's open source tool for creating identical machine images for multiple platforms from a single source configuration. The machine image created here can be customized through modifications to the [build configuration file](packer.json) and the [shell script](../shared/scripts/setup.sh).
```

Comment:
This does not render correctly in the tutorial; it shows up as "modifications to the and the." I was not sure about the intended destination of the links; should these should be  file:/// URLs or just not markdown? :)

Reply:
I fixed the links with the full URL paths to each resource. That should fix the rendering problem on cloud shell. But, the `build configuration file` link in the cloud shell tutorial won't work until this PR is merged to the default branch. It should render fine now.